### ISSUE TITLE
[TOGZHAN | MADELINE | FAHIM] High Fidelity Sign-in Page & Forgot Password Support

### ISSUE THREAD

**Issue by lillian-tran** [JumboCode/dillar-academy#54]
### Task Description: 
We’re making the sign-in forms ✨prettier.✨
You’ll be working on the high fidelity versions of the login and signup pages. They are both pretty similar so it’s a matter of standardizing fonts, colors, etc. Don't worry about making the footer - this is part of someone else's ticket.
Try to make the sign-up page look as close as this as possible: 
<img width="531" alt="Screenshot 2024-11-23 at 2 30 59 PM" src="https://github.com/user-attachments/assets/0c1e0a97-5c9b-410d-905a-e8aee4faf8f0">
_Two slight differences: ignore the forgot password here and also leave the password requirements even if it's not on the Figma._

Then, try to make the log-in page as close to this as possible:  
<img width="525" alt="Screenshot 2024-11-23 at 2 31 54 PM" src="https://github.com/user-attachments/assets/5f282122-daaf-4f6e-97c4-1492dea35323">
This time do not ignore the “forgot password” on this page! We have to actually implement it now though! 🙈
Create a new page in ForgotPassword.jsx that has a form and takes input for the user’s username, new password, and retyped password to confirm. You should use the Form, FormInput, FormSubmit, and PasswordReqs components to create the form, which means the form styling is already done for you (yay components)!  There’s no design for the forgot password page but you can style it to look like the Login and Sign up pages where the background is a gradient and the form is centered on the page. Once the forgot password page is done, we can now actually implement the functionality! Create an endpoint that takes in the form data to get the username and password and update the password associated with the user in the database.

### Acceptance Criteria: 
- [ ] Did you link the ticket/issue number with your branch using the pull request template?
- [ ] Does submitting the Forgot Password form update the password in the database?
- [ ] Does the Forgot Password page look similar to the Sign up and Login pages?

### References and Resources:
[Figma](https://www.figma.com/design/bQkMH2fS2NAYoWDlynDh1W/Dillar-Academy?node-id=1-1771&node-type=canvas&t=WqQL3joCTYiIjf3o-0 )
MongoDB collection commands: https://www.mongodb.com/docs/manual/reference/method/js-collection/ 
Mongo Updating Documents: https://www.mongodb.com/docs/manual/tutorial/update-documents/
